lib/core/versions/latest/models/AnchorFileModel.ts

```diff
@@ -6,7 +6,10 @@ export default interface AnchorFileModel {
   map_file_uri: string;
   operations: {
     create?: {
-      suffix_data: string;
+      suffix_data: {
```

Comment:
seems like this type should be imported from somewhere common.

Reply:
This structure is not used by anything else so I think leaving it here makes sense. We can pull it out when there's use case for it in the future.